api/models.py

```diff
@@ -125,6 +126,7 @@ class UserProfile(BaseModel):
         default=[],
         description="A list of groups that user belongs to"
     )
+    email: EmailStr
```

Comment:
Please define it as a `Field` with a description so that appears in the auto-generated documentation.